src/blocks/media-block/components/save.js

```diff
@@ -0,0 +1,67 @@
+import { Component } from "@wordpress/element";
+import cls from 'classnames';
+
+export class MediaBlockSaveComponent extends Component{
+
+    render () {
+
+        const { attributes } = this.props;
+
+        const isLinkAvailable = attributes.url !== undefined && attributes.url !== "";
+
+        let titleText = isLinkAvailable ?
+            <a href={ attributes.url } className={"c-block__title-link u-theme--link-hover--dark"}> {attributes.title} </a> :
+            attributes.title;
+
+        const styles = cls(
+            'c-media-block c-block',
+            {'c-block__row': attributes.alignment === "left"},
+            {'c-block__stacked': attributes.alignment === "center"}
+        );
+
+        return (
+            <div className={styles}>
```

Comment:
Depending on if the sidebar is on or off, you will need to add the following classes:

**On:**
`l-grid--7-col l-grid-wrap--3-of-7`

**Off:**
`l-grid--7-col l-grid-wrap--4-of-7`

Reply:
Thank you, but this should be configured not in this ticket, but in the theme itself. Thank you for the comment, I will also fix it in the theme

Reply:
This is classes that need to be applied in this block, not the theme.

Reply:
Thanks